Source/Utils/WarpXUtil.cpp

```diff
@@ -213,3 +213,15 @@ WarpXParser makeParser (std::string const& parse_function, std::vector<std::stri
     }
     return parser;
 }
+
+namespace WarpXUtilMsg{
+
+void AlwaysAssert(int expression, const std::string& msg = "ERROR!")
```

Comment:
```suggestion
void AlwaysAssert(bool expression, const std::string& msg = "ERROR!")
```

Reply:
I agree... I was in doubt between `int` and `bool` and I finally chose to mimic the `assert` function in the C standard library. However, on a second though, `bool` is better...